account_statement_import_base/README.rst

```diff
@@ -0,0 +1 @@
+Will be autogenerated from the readme subdir
```

Comment:
@pedrobaeza would really the absence of the file trigger a red CI?

Reply:
It can be